pkg/sig/signature_manager.go

```diff
@@ -0,0 +1,50 @@
+package sig
```

Comment:
Can you rename this folder/package to signature. The name `sig`, is fairly generic